src/Aspire.Hosting/ApplicationModel/ContainerLifetimeAnnotation.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace Aspire.Hosting.ApplicationModel;
+
+/// <summary>
+/// Lifetime modes for container resources
+/// </summary>
+public enum ContainerLifetimeType
+{
+    /// <summary>
+    /// The resource is tied to the lifetime of the AppHost.
+    /// </summary>
+    AppHost,
```

Comment:
```suggestion
    Default
```

Reply:
Does it make sense to have both `Default` and `AppHost`? Obviously `Default` would currently be the same as `AppHost`, but if we foresee a potential future where the meaning of `Default` could change, users may want to explicitly opt-out by keeping a resource in `AppHost` lifetime mode.

Reply:
I think default without apphost

Reply:
Should this be a [Flags]?

Reply:
Seems like the values would be mutually exclusive; is there any particular benefit to adding `[Flags]` on an Enum with exclusive values?

Reply:
I don't think so.